relayertest/test.go

```diff
@@ -187,15 +187,16 @@ func TestRelayer(t *testing.T, cf ibctest.ChainFactory, rf ibctest.RelayerFactor
 	)
 
 	for _, testCaseConfig := range relayerTestCaseConfigs {
-		if len(missingCapabilities(rf, testCaseConfig.RequiredRelayerCapabilities...)) > 0 {
-			continue
-		}
```

Comment:
This was the culprit. By bailing out here, we do not add the test caste to the testCases slice on line 196. Thus the test is omitted entirely. 

We have code later that `t.Skip`s tests.